src/components/Navigation/UserInformation.tsx

```diff
@@ -33,7 +33,7 @@ export const UserInformation: React.FC<{}> = () => {
       <div className={useStyles().container}>
         {profile.value == null ? (
```

Comment:
nit: use `!!` to remove a `==` check?
```suggestion
        {!!profile.value ? (
```

Reply:
oops 😅 yeah, that's what I meant!